### ISSUE TITLE
util.parse_url doesnt parse path when schema https: or http: etc is missing

### ISSUE THREAD

**Issue by appunni-dishq** [urllib3/urllib3#1539]
//chuaochocolatier.com/chocolate-bars/types-of-chocolate/milk-chocolate.html

Example urllib parsing:

```python
>>> from urllib import parse
>>> parse.urlparse("//chuaochocolatier.com/chocolate-bars/types-of-chocolate/milk-chocolate.html")
ParseResult(scheme='', netloc='chuaochocolatier.com', path='/chocolate-bars/types-of-chocolate/milk-chocolate.html', params='', query='', fragment='')
```

Example urllib3 parsing:

```python
>>> from urllib3.util import parse_url
>>> parse_url("//chuaochocolatier.com/chocolate-bars/types-of-chocolate/milk-chocolate.html")
Url(scheme=None, auth=None, host=None, port=None, path='//chuaochocolatier.com/chocolate-bars/types-of-chocolate/milk-chocolate.html', query=None, fragment=None)
```

I suppose this is a bug. Please let me know how I can help.

**Comment by sethmlarson** [comment]
Is this against master or the latest released version? If not against master could you try again using the latest URL parsing code?

**Comment by sethmlarson** [comment]
This is closed via #1531
